CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ message(STATUS "Version: ${VERSION_TAG}")
 
 add_definitions("-DVERSION=${VERSION_TAG}")
 
-set(CMAKE_CXX_STANDARD 14)
```

Comment:
Ok. That was just for the syntactic sugar.

Reply:
Can you put it back to CMAKE_CXX_STANDARD 14, as it was previously? It may cause issues otherwise when C++14 is greater than the default for a toolchain (sorry, I know this is possibly overly conservative, but I've fluted about with moving this back and forth, and it always bites)

Reply:
Done.